pytest_fixtures/core/sat_cap_factory.py

```diff
@@ -321,7 +331,9 @@ def cap_ready_rhel():
         'workflow': settings.capsule.deploy_workflows.os,
     }
     with Broker(**deploy_args, host_class=Capsule) as host:
+        host.enable_ipv6_http_proxy()
         yield host
+        host.disable_ipv6_http_proxy()
```

Comment:
I would hold off the answer a bit because, I am interested to know about whether Teams really want to teardown it since we have components split CI jobs. 

Also, the disable is written to disable based on the needs per test/module/component for now, I just added it everywhere to show Teams how it looks like when implemented.

Reply:
@ogajduse I am not going to hold it anymore for discussion with teams as its not really useful , I recall that its us / JPL who should decide it. So I would like to continue the discussion on this comment.

Reply:
@ogajduse Any further thohghts before I resolve the comment as is ?

Reply:
I'm also interested in moving this to setup/teardown methods for the Host class to reduce the need for repetition here.

Reply:
As discussed yesterday in 1:1, having a teardown is not necessary as once the test execution is done on the satellite (be its xdist sat or destructive sat), the satellite will be either checked-in OR even if it kept for debugging there is no harm in keeping that satellite proxy enabled.

There are few components could be worried about keeping pre-enabled proxy and that is `HTTP Proxy` component itself and which has its own custom HTTP proxy servers (auth, unauth etc). That component can disable the default HTTP-proxy during the component setup phase or checkout its own on-demand sat. I would leave that up to the component team to decide how do they want to tackle it.

Overall, I feel like there is no need of explicit teardown here unless asked explicitly.